packages/sdk/src/modules/unification/index.ts

```diff
@@ -0,0 +1,118 @@
+import { Base } from '../base';
+import { ApiPromise } from '@polkadot/api';
+import { decodeAddress } from '@polkadot/keyring';
+import type { ISubmittableResult } from '@polkadot/types/types';
+
+import type { SDKMetadata } from '../../types';
+import { ethers, Wallet } from "ethers";
+
+interface ClaimAccountOptions {
+  substrateSeed: string,
+  ethPrivate: string
+}
+
+export class Unification extends Base {
+    constructor(
+      protected override readonly _api?: ApiPromise,
+      protected readonly _metadata?: SDKMetadata,
+  
+    ) {
+      super();
+    }
+
+  // TODO just return the ethereum address?
+  // How does the user know what their mnenonmic phrase and private key are...
+  //  - Can we safely return??
+
+  public async claimAccount(options: ClaimAccountOptions, statusCallback?: (result: ISubmittableResult) => void | Promise<void>): Promise<string> {
```

Comment:
Currently only allowing newly created eth wallets to be mapped to a SS58 address.